service/templates/distros.html

```diff
@@ -887,6 +887,11 @@ <h2 style="display:inline-block; padding-right:15px">Configure
                 Expansion</button>
             </div>
           </div>
+          <div>
+            <label class="distro-label">Valid Projects:</label>
+            <textarea ng-readonly="readOnly" name="valid_projects" ng-list="&#10;" class="form-control" rows="5"
```

Comment:
This needs ng-trim="false" otherwise each character will be considered a separate project

Reply:
done